app_dart/lib/src/request_handlers/github/webhook_subscription.dart

```diff
@@ -113,10 +113,16 @@ class GithubWebhookSubscription extends SubscriptionHandler {
         // We'll leave unfinished jobs if it was merged since we care about those
         // results.
         if (!pr.merged!) {
-          await scheduler.cancelPreSubmitTargets(pullRequest: pr, reason: 'Pull request closed');
+          await scheduler.cancelPreSubmitTargets(
```

Comment:
Is there a point to have the if/else logic contain cancelPresubmitTargets?

You can move it above to do it regardless, and if the reason is useful, do `reason: 'Pull request ${pr.merged ? 'merged' : 'closed'}'`

Reply:
No. I will update the code.